docs/developer/metadata.mdx

```diff
@@ -70,8 +67,36 @@ Here's an example query that fetches the metadata of the first product:
       }
     }
   }
-}`}
-/>
+}
+```
+
+Response:
+```json
+{
+  "data": {
+    "products": {
+      "edges": [
+        {
+          "node": {
+            "id": "UHJvZHVjdDo3Mg==",
+            "name": "Apple Juice",
+            "metadata": [
+              {
+                "key": "vatlayer.code",
+                "value": "standard"
+              },
+              {
+                "key": "vatlayer.description",
+                "value": "standard"
+              }
+            ]
```

Comment:
I think we can find better examples of metadata?

Reply:
IMO it's a good example of using metadata for app synchronization. @patrys  Removing description would make this better? (removed duplication)

Reply:
I was thinking about using something self-explanatory like `pim.externalReference` and `tax.group` maybe?